### PR TITLE
EPMRPP-33148 React. LogItemInfo panel

### PR DESCRIPTION
**Pull request by ultd39**
LogItemInfo panel implementation.

### CODE REVIEW

File: app/src/pages/inside/logsPage/logsPage.jsx [hunk 1]
```diff
@@ -2,25 +2,42 @@ import { Component } from 'react';
...
+
+  static defaultProps = {
+    activeItemId: '',
+    historyItems: [],
   };
 
   render() {
-    const { refresh } = this.props;
+    const { refresh, activeItemId, historyItems } = this.props;
+    const activeLogItem = historyItems.find((historyItem) => historyItem.id === activeItemId);
```
Comment (ultd39) [inline app/src/pages/inside/logsPage/logsPage.jsx]: I'm not sure about getting active item. Maybe for it better to use some selector but I didn't find it.

File: app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  transition: fill 0.2s, color 0.2s;
+  height: 40px;
+  box-sizing: border-box;
+  background-color: transparent;
+  border: 0;
+  outline: 0;
+
+  &:hover {
+    color: #000;
+    fill: #000;
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss]: On our project all colors should be defined with color variables from color palette. If you see the color which is not represented in color palette - discuss it with our UX Designer.

File: app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    width: 20px;
+    height: 20px;
+  }
+}
+
+.tab-label {
+  font-size: 13px;
+  text-transform: uppercase;
+  font-family: $FONT-SEMIBOLD;
+  font-weight: bold;
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss]: Please do not use font-weight property. We have font types for all required weights.

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    defaultMessage: 'History of actions',
+  },
+});
+
+const makeTabs = ({ formatMessage }) => [
+  {
+    id: 'stack',
+    label: formatMessage(messages.stackTab),
+    getIcon: () => Parser(StackTraceIcon),
+    getContent: () => <div>Stack trace</div>,
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx]: why functions?

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      <div className={cx('container')}>
+        {logItem && (
+          <Fragment>
+            <div className={cx('content')}>
+              <div className={cx('description')}>
+                {logItem.issue && <DefectType issue={logItem.issue} />}
+              </div>
+              <div className={cx('actions')}>
+                <div className={cx('action')}>
+                  <GhostButton icon={DownLeftArrowIcon}>
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx]: I think it's a good idea to disable buttons with not implemented actions. @ArtsiomYeliseyenka what do you think?

File: app/src/pages/inside/logsPage/logsPage.jsx (hunk 1 above)
Comment (DiscoElevator) [reply app/src/pages/inside/logsPage/logsPage.jsx]: I think it makes sense to add a selector for this

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx (hunk 5 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx]: +1
I think we will get a less questions related to errors in console from our QA team in that case.

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx (hunk 4 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx]: Also Parser function now calling inside of InfoTabs component.

File: app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss (hunk 2 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss]: Discussing with designers...

Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss]: We decided that we should use $COLOR--charcoal-grey here.

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx (hunk 5 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx]: Agree with you. I disabled this buttons.

File: app/src/pages/inside/logsPage/logsPage.jsx (hunk 1 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logsPage.jsx]: Maked activeLogSelector.

**Comment by ultd39** [comment]
In some cases activeLogItem becomes undefined and we can't render LogItemInfo so I added check for that and marked logItem as required.

**Comment by ultd39** [comment]
Okay, I fixed all of issues.

File: app/src/pages/inside/logsPage/logItemInfo/InfoTabs/InfoTabs.jsx [hunk 6]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Parser from 'html-react-parser';
 import ArrowDownIcon from 'common/img/arrow-down-inline.svg';
-import styles from './logItemInfoTabs.scss';
+import styles from './InfoTabs.scss';
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/InfoTabs/InfoTabs.jsx]: Oh, just one more point we've missed: `InfoTabs` -> `infoTabs` - we use camelCase for file names.

Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/InfoTabs/InfoTabs.jsx]: Yes, right. Fixed.

**Pull request merged**